web/source/osk/oskManager.ts

```diff
@@ -482,17 +470,25 @@ namespace com.keyman.osk {
     }
 
     /**
-     * Move OSK back to default position
+     * Function     restorePosition
+     * Scope        Public
+     * Description  Move OSK back to default position, floating under active input element
      */
     restorePosition: () => void = function(this: OSKManager) {
```

Comment:
```suggestion
    ['restorePosition']: () => void = function(this: OSKManager) {
```

Reply:
Without this change, we'll lose access to the function in minified versions of KMW.